samples/authentication/serial/auth_server/sample.yaml

```diff
@@ -0,0 +1,8 @@
+sample:
+  description: Auth Server serial transport sample app
+  name: Auth Server Serial
+tests:
+  sample.auth_server_serial:
+    harness: bluetooth
```

Comment:
`bluetooth` harness probably isn't correct here, but without it I get a timeout error: `INFO    - 1/1 qemu_x86                  samples/authentication/serial/auth_server/sample.auth_server_serial FAILED Timeout (qemu)`

Any guidance on how to correctly use the harness option is appreciated.